tuning/automation/rocblas-benchInputCreator.py

```diff
@@ -158,12 +178,12 @@ def createYaml(args, problem, sizeMappings, verify):
         otherParams = {"alpha": 1, "beta": 1, "iters": 10, "cold_iters": 2}
 
     # create rocBLAS-bench call for each size in logic file
-    for (size, _) in sizeMappings: # size[0] = M, size[1] = N, size[2] = batch_count, size[3] = K
+    for (size, _) in sizeMappings: # size[0] = M, size[1] = N, size[2] = batch_count, size[3] = K, size[4] = ldc, size[5] = ldd, size[6] = lda, size[7] = ldb
         params = {}
  
-        if (size[2] == 1 and not f8gemm):  # non-f8, non-batched gemm (severes both HPA and non-HPA)
+        if (size[2] == 1 and not f8gemm):  # non-f8, non-batched gemm (serves both HPA and non-HPA)
             params["rocblas_function"] = "rocblas_gemm_ex"
-        elif (size[2] != 1 and not f8gemm): # non-f8, strided_batched gemm (severes both HPA and non-HPA)
+        elif (size[2] != 1 and not f8gemm): # non-f8, strided_batched gemm (serves both HPA and non-HPA)
```

Comment:
Don't we support StridedBatched=False?

Reply:
I have not done any task for stridedbatched false. I will add the feature, when I get the first task.